aspnetcore/fundamentals/servers/httpsys/sample/HttpSysDemo.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AssemblyName>HttpSysDemo</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageId>sample</PackageId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Update="wwwroot\**\*">
```

Comment:
Yes. Sdk.Web was effective with *wwwroot*. There were only edge cases where someone had to hack it for certain types of files: For example if it tried to move something that you didn't want moved, you could stop it from doing so. If it wouldn't move something that you wanted moved, you bend it to your will and force it to move the file.

In these samples, I don't think it serves any purpose: There is either no *wwwroot* folder in the sample (nothing will happen), or there's an *demo.html* file (that will be moved by Sdk.Web).